ts/Accessibility/AccessibilityComponent.ts

```diff
@@ -342,8 +342,8 @@ AccessibilityComponent.prototype = {
             }, attributes);
 
         Object.keys(attrs).forEach(function (prop: string): void {
-            if (attrs[prop] !== null) {
-                proxy.setAttribute(prop, attrs[prop]);
+            if ((attrs as any)[prop] !== null) {
+                proxy.setAttribute(prop, (attrs as any)[prop]);
```

Comment:
If you set the `attrs` variable to the type `(SVGAttributes&Record<string, any>)`, you do not have to make `any` casts. On the other hand I like to see the Record<string, any> type from all the place removed as well. So maybe we should introduce in the global namespace of `Core/Globals` a type alias `type AnyRecord = Record<string, any>`. So `attrs` should then get the type `(SVGAttributes&AnyRecord)`. What do you think about that idea? Would this work for all the other new `any` casts that pop up?

Reply:
It looks like in this case the problem was actually that `attrs` can be undefined.

Apart from that, I think most of those new `any`s have popped up in situations where we are looping objects. It looks like it can be resolved by using `keyof`. Should we go with that solution? Then we preserve the strict typing.
```ts
        Object.keys(attrs || {}).forEach(function (prop: keyof HTMLAttributes): void {
            if (attrs && attrs[prop] !== null) {
                groupDiv.setAttribute(prop, String(attrs[prop]));
            }
        });
```